programs/degen-pools/src/pools.rs

```diff
@@ -19,6 +19,12 @@ pub struct PoolCreated {
     pub description: String,
 }
 
+#[event] 
+pub struct PoolPaused {
+    pub isPaused: bool,
```

Comment:
Are you using VSCode for development? If so, I recommend installing the [rust-analyzer extension](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer). The default editor support is not that great.

Reply:
Yes, I had already installed rust-analyzer as you suggested.
I was initially using camelCase as mentioned in the ticket, likely because both backend tickets had the same naming convention.
 However, it would be better to use snake_case as per the recommendation.
 I hope we can align this convention in the backend as well.

Reply:
Ohh okay. Yeah it's tricky how the casing convention shifts. It seems anchor enforces snake case in rust, and camel case in JS. Perhaps it's easiest just to follow their standard.